.azure-pipelines.yml

```diff
@@ -0,0 +1,45 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-16.04'
+
+strategy:
+  matrix:
+    Node6:
+      NODE_VERSION: '6.x'
+      TEST_TITLE: 'Node 6 JUnit Test Results'
```

Comment:
I would just build the "test title" value using the `NODE_VERSION` variable (and drop all the `TEST_TITLE` matrix variables) -- this will cut down on the noise ..

```yaml
  - task: PublishTestResults@2
    inputs:
      testResultsFormat: JUnit
      testResultsFiles: './test-results.xml'
      testRunTitle: '$(NODE_VERSION) Tests`
```